mobile/src/Teacher/screens/Explore/DetailScreen.js

```diff
@@ -0,0 +1,118 @@
+import React, { Fragment, useState } from 'react';
+import NavBarView from './NavBarView'
+import { fontFamilies, fonts } from '../../../utils/theme'
+import { FlatList } from 'react-native-gesture-handler'
+import {
+    StyleSheet,
+    Text,
+    SafeAreaView,
+    View,
+    Button,
+    TouchableOpacity,
+    Pressable
+} from 'react-native';
+import { inOut } from 'react-native/Libraries/Animated/src/Easing';
+import Card from './QuestionCard';
+import { createStackNavigator } from '@react-navigation/stack';
+import ButtonBack from '../../../components/ButtonBack/index';
+import Aicon from 'react-native-vector-icons/FontAwesome';
+import LinearGradient from 'react-native-linear-gradient'
+
+// import GameDetailsScreen from './GameDetails';
+// import Games from '../Games';
+
+const detailList = ({route, navigation}) => {
+
+    const { game } = route.params;
+
+    const questions = [game.q1,game.q2,game.q3,game.q4,game.q5].filter( (question) => !!question ) 
```

Comment:
What's `!!`? Is that "not not" or a js operator?

Reply:
Yes, `!!` is a js operator  for "not not". I think the purpose of that is to check if there are 5 questions or not through boolean.